draft-ietf-tls-subcerts.md

```diff
@@ -235,7 +236,7 @@ protocol is not allowed.
 Delegated credentials allow a peer to terminate TLS connections on behalf of
 the certificate owner.  If a credential is stolen, there is no mechanism for
 revoking it without revoking the certificate itself.  To limit exposure in case
-of delegated credential private key compromise, delegated credentials have a maximum
+of delegated credential's private key compromise, delegated credentials have a maximum
```

Comment:
How about "of the compomise of a delegated credential's private key, delegated credentials have a ..."